dist/licenses.txt

```diff
@@ -0,0 +1,463 @@
+@actions/core
```

Comment:
We need to make 100% sure that we attribute all authors and reproduce the license wherever the license requires it. Can we rely on `ncc` to work correctly?

Reply:
FTR: we went over the output of `ncc` and agreed that (for now) it covers what we would expect it to cover.